src/components/DisplayPanel.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function DisplayPanel(props) {
+  const { res } = props;
+  const { next, total, history } = res;
+  const disRes = next || total;
+  return (
+    <div className="disContainer">
+      <p className="history">{history}</p>
+      <p>{disRes}</p>
+    </div>
+  );
+}
+
+// the default value
+DisplayPanel.defaultProps = {
+  res: '0',
+};
```

Comment:
- remember, this default value had to appear on the calculator screen (from `milestone 2`, I will like you to make it display
For now, your calculator launches with a BLANK screen